esignet-mosipid.properties

```diff
@@ -109,36 +109,36 @@ mosip.esignet.captcha-validator.secret=${esignet.captcha.secret.key}
-mosip.esignet.ida.vci-exchange-version=1.0
-mosip.esignet.ida.vci-exchange-url=https://${mosip.api.internal.host}/idauthentication/v1/vci-exchange/delegated/${mosip.esignet.misp.license.key}/
+#mosip.esignet.authenticator.ida-auth-id=mosip.identity.kycauth
+#mosip.esignet.authenticator.ida-exchange-id=mosip.identity.kycexchange
+#mosip.esignet.authenticator.ida-send-otp-id=mosip.identity.otp
+#mosip.esignet.authenticator.ida-version=1.0
+#mosip.esignet.authenticator.ida-domainUri=https://${mosip.esignet.mosipid.host}
+#mosip.esignet.authenticator.ida.cert-url=https://${mosip.api.public.host}/mosip-certs/ida-partner.cer
+#mosip.esignet.authenticator.ida.kyc-auth-url=https://${mosip.api.internal.host}/idauthentication/v1/kyc-auth/delegated/${mosip.esignet.misp.license.key}/
+#mosip.esignet.authenticator.ida.kyc-exchange-url=https://${mosip.api.internal.host}/idauthentication/v1/kyc-exchange/delegated/${mosip.esignet.misp.license.key}/
+#mosip.esignet.authenticator.ida.send-otp-url=https://${mosip.api.internal.host}/idauthentication/v1/otp/${mosip.esignet.misp.license.key}/
+#mosip.esignet.binder.ida.key-binding-url=https://${mosip.api.internal.host}/idauthentication/v1/identity-key-binding/delegated/${mosip.esignet.misp.license.key}/
+#mosip.esignet.authenticator.ida.get-certificates-url=https://${mosip.api.internal.host}/idauthentication/v1/internal/getAllCertificates
+#mosip.esignet.authenticator.ida.auth-token-url=https://${mosip.api.internal.host}/v1/authmanager/authenticate/clientidsecretkey
+#mosip.esignet.authenticator.ida.audit-manager-url=https://${mosip.api.internal.host}/v1/auditmanager/audits
+#mosip.esignet.authenticator.ida.client-id=mosip-ida-client
+#mosip.esignet.authenticator.ida.secret-key=${mosip.ida.client.secret}
+#mosip.esignet.authenticator.ida.app-id=ida
+#mosip.esignet.authenticator.ida-env=Developer
+#mosip.esignet.authenticator.ida.otp-channels=email,phone
+
+#mosip.esignet.ida.vci-user-info-cache=userinfo
+#mosip.esignet.ida.vci-exchange-id=mosip.identity.vciexchange
+#mosip.esignet.ida.vci-exchange-version=1.0
+#mosip.esignet.ida.vci-exchange-url=https://${mosip.api.internal.host}/idauthentication/v1/vci-exchange/delegated/${mosip.esignet.misp.license.key}/
 
 # Mock IDA integration props
-mosip.esignet.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
-mosip.esignet.mock.authenticator.kyc-auth-url=https://${mosip.api.public.host}/v1/mock-identity-system/kyc-auth
-mosip.esignet.mock.authenticator.kyc-exchange-url=https://${mosip.api.public.host}/v1/mock-identity-system/kyc-exchange
+mosip.esignet.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system-mosipid/identity
+mosip.esignet.mock.authenticator.kyc-auth-url=https://${mosip.api.public.host}/v1/mock-identity-system-mosipid/kyc-auth
+mosip.esignet.mock.authenticator.kyc-exchange-url=https://${mosip.api.public.host}/v1/mock-identity-system-mosipid/kyc-exchange
```

Comment:
We should not change the context-path, instead domain name should be updated with service name w.r.t namespace it is deployed.

Reply:
removed mosipid